.gitlab/kitchen_deploy/kitchen_deploy.yml

```diff
@@ -121,8 +107,6 @@ deploy_deb_testing-a7_x64:
     [
       "agent_deb-x64-a7",
       "agent_heroku_deb-x64-a7",
-      "iot_agent_deb-x64",
-      "dogstatsd_deb-x64",
       "lint_linux-x64",
     ]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** Is this job still necessary‏

Reply:
I'm not sure to understand why there are still -a7 jobs like these

Reply:
The "remove a7" part is handled on the other PR #30261. The goal of this one was first to remove what we are not supposed to build (dogstastsd, iot agent, installer, dmg and some windows publications)